### PR TITLE
1522 html example text

### PR DESCRIPTION
**Pull request by hellokitties**
#1522 
added example for text element https://jdi-docs.github.io/jdi-light/?java#text

### CODE REVIEW

File: jdi-light-html-tests/src/test/java/io/github/epam/example/common/TextExampleTests.java [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+        jdiText.is()
+                .enabled()
+                .and()
+                .displayed()
+                .core()
+                .css("font-size", is("14px"));
+    }
+
+    @Test
+    public void textEqualsTest() {
```
Comment (OBraunEPAM) [inline jdi-light-html-tests/src/test/java/io/github/epam/example/common/TextExampleTests.java]: Suggest to rename this test for better understanding
```suggestion
    public void textPositiveTest() {
```

File: jdi-light-html-tests/src/test/java/io/github/epam/example/common/TextExampleTests.java [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+        jdiText.is().text(TEXT);
+    }
+
+    @Test
+    public void textContainsTest() {
+        jdiText.is().text(containsString(PART_OF_TEXT));
+    }
+
+    @Test
+    public void textNotContainsWordTest() {
```
Comment (OBraunEPAM) [inline jdi-light-html-tests/src/test/java/io/github/epam/example/common/TextExampleTests.java]: Suggest to rename this test for better understanding
```suggestion
    public void textDoesNotContainWordTest() {
```

File: jdi-light-html-tests/src/test/java/io/github/epam/example/common/TextExampleTests.java (hunk 1 above)
Comment (hellokitties) [reply jdi-light-html-tests/src/test/java/io/github/epam/example/common/TextExampleTests.java]: fixed

Comment (hellokitties) [reply jdi-light-html-tests/src/test/java/io/github/epam/example/common/TextExampleTests.java]: renamed

File: jdi-light-html-tests/src/test/java/io/github/epam/example/common/TextExampleTests.java (hunk 2 above)
Comment (hellokitties) [reply jdi-light-html-tests/src/test/java/io/github/epam/example/common/TextExampleTests.java]: renamed

**Pull request merged**